jitbuilder/release/cpp/samples/Mandelbrot.cpp

```diff
@@ -340,6 +339,12 @@ main(int argc, char *argv[])
 
    printf("Step 5: output result buffer\n");
    FILE *out = (argc == 3) ? fopen(argv[2], "wb") : stdout;
+
+   // The file we generate here is in the PPM image format which has a simple header describing how to interpret the
+   // buffer. This string needs to be encoded as ASCII in the file generated, however on platforms such as z/OS whose
+   // native encoding is not ASCII this is problematic.
+   //
+   // TODO (#4901): Use the port library to output the header bytes for PPM image file format
```

Comment:
Linking this PR with #4901 so GitHub notes the link for future whenever we fix this.